rfd/0196-opinionated-intro.md

```diff
@@ -0,0 +1,150 @@
+---
+authors: Dave Sudia (david.sudia@goteleport.com)
+state: draft
+---
+
+# RFD 0196 - Opinionated Intro to the Product
+
+## Overview
+
+### Problem
```

Comment:
For scope of this RFD, are we thinking about it for Teleport Cloud, Teleport Community, and Teleport Enterprise.

Reply:
I think Cloud and Enterprise since they would have more feature parity, less checks to worry about on what to display. Unless it's just as easy to have it in community as well.